qiskit_nature/second_q/properties/heisenberg_model.py

```diff
@@ -0,0 +1,113 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Heisenberg model."""
+
+import logging
+from typing import Optional, Tuple
+from fractions import Fraction
+import numpy as np
+from qiskit_nature.second_q.operators import SpinOp
+from qiskit_nature.second_q.properties.lattice_model import LatticeModel
+from qiskit_nature.second_q.properties.lattices.lattice import Lattice
+
+logger = logging.getLogger(__name__)
+
+
+class HeisenbergModel(LatticeModel):
+    """The Heisenberg model."""
+
+    def __init__(
+        self,
+        lattice: Lattice,
+        coupling_constants: Tuple = (1.0, 1.0, 1.0),
+        ext_magnetic_field: Tuple = (0.0, 0.0, 0.0),
+    ) -> None:
+        """
+        Args:
+            lattice (Lattice): Lattice on which the model is defined.
+            coupling_constants (Tuple, optional): The coupling constants in each Cartesian axes.
+                                                  Defaults to (1.0, 1.0, 1.0).
+            ext_magnetic_field (Tuple, optional): Represents a magnetic field in Cartesian coordinates.
```

Comment:
The types in the args here are not needed - they duplicate the info that is  already in the typehints on the parameters which Sphinx will pick up on - so these should all be removed leaving just the parameter name: description